addons/binding/org.openhab.binding.samsungtv/ESH-INF/i18n/samsung_de.properties

```diff
@@ -0,0 +1,47 @@
+# binding
+binding.samsungtv.name = Samsung TV Binding
+binding.samsungtv.description = Dieses Binding integriert Samsung TV Ger�te. Durch dieses k�nnen Samsung TV Ger�te gesteuert werden.
+
+# thing types
+thing-type.samsungtv.tv.label = Samsung TV
+thing-type.samsungtv.tv.description = Samsung TV. Dient zur Steuerung des Ger�tes und liefert Daten wie z.B. Infos �ber den aktuellen Kanal oder die laufende Sendung.
+
+# thing types config
+thing-type.config.samsungtv.tv.hostName.label = IP-Adresse
+thing-type.config.samsungtv.tv.hostName.description = Lokale IP-Adresse oder Hostname des Samsung TV.
+thing-type.config.samsungtv.tv.port.label = Port
+thing-type.config.samsungtv.tv.port.description = Port des Samsung TV.
+thing-type.config.samsungtv.tv.refreshInterval.label = Abfrageintervall
+thing-type.config.samsungtv.tv.refreshInterval.description = Intervall zur Abfrage des Samsung TV (in Millisekunden).
+
+# channel types
+channel-type.samsungtv.volume.label = Lautst�rke
+channel-type.samsungtv.volume.description = Erm�glicht die Steuerung der Lautst�rke.
+channel-type.samsungtv.mute.label = Stumm schalten
+channel-type.samsungtv.mute.description = Erm�glicht die Lautst�rke auf stumm zu schalten.
+channel-type.samsungtv.brightness.label = Helligkeit
+channel-type.samsungtv.brightness.description = Erm�glicht die Steuerung der Helligkeit.
+channel-type.samsungtv.contrast.label = Kontrast
+channel-type.samsungtv.contrast.description = Erm�glicht die Steuerung des Kontrast.
+channel-type.samsungtv.sharpness.label = Sch�rfe
+channel-type.samsungtv.sharpness.description = Erm�glicht die Steuerung der Sch�rfe.
+channel-type.samsungtv.colortemperature.label = Farbtemperatur
+channel-type.samsungtv.colortemperature.description = Erm�glicht die Steuerung der Farbtemperatur. Von 0 bis 4.
+channel-type.samsungtv.sourcename.label = Source Name
```

Comment:
Should this be translated, too?
Something like `Eingangsname` or `Quellenname` maybe?
But that does not sound right, either. 🤔

Reply:
If translating this I would suggest `Eingangsquelle`. Or just `Eingang`. wdyt?